packages/FormElement/src/FormElement.js

```diff
@@ -125,23 +131,30 @@ function FormElement(props) {
     });
   }
   return (
-    <div css={formElementStyles} isInline={isInline} size={size} isDisabled={isDisabled} {...moreProps}>
+    <sc.FormElement
+      as={hasFieldSet ? "fieldset" : "div"}
```

Comment:
Outer container now rendering as a `fieldset` element if `isFieldSet` prop is true